modules/standard/timers/timer_controller.py

```diff
@@ -63,9 +64,10 @@ def timer_list_cmd(self, request):
             repeats = (" (Repeats every %s)" % self.util.time_to_readable(timer.repeating_every)) if timer.repeating_every > 0 else ""
             blob += "<pagebreak>Name: <highlight>%s</highlight>\n" % timer.name
             blob += "Time left: <highlight>%s</highlight>%s\n" % (self.util.time_to_readable(timer.created_at + timer.duration - t, max_levels=None), repeats)
-            blob += "Owner: <highlight>%s</highlight>\n\n" % timer.char_name
-
-        return ChatBlob("Timers (%d)" % len(data), blob)
+            blob += "Owner: <highlight>%s</highlight>\n" % timer.char_name
+            blob += "%s" % self.text.make_tellcmd ("Remove", "timer remove %s") % timer.name + "\n\n"
```

Comment:
I think this is the line you want:
`blob += self.text.make_tellcmd("Remove", "timer remove %s" % timer.name) + "\n\n"`